src/src/main/java/org/redfrog404/mobycraft/entity/EntityChaosMonkey.java

```diff
@@ -287,13 +276,13 @@ public boolean getCanSpawnHere() {
 				return false;
 			}
 
-			return i > this.rand.nextInt(j) ? false : super.getCanSpawnHere();
+			return i <= this.rand.nextInt(j) && super.getCanSpawnHere();
 		}
 	}
 
 	private boolean isDateAroundHalloween(Calendar p_175569_1_) {
-		return p_175569_1_.get(2) + 1 == 10 && p_175569_1_.get(5) >= 20
-				|| p_175569_1_.get(2) + 1 == 11 && p_175569_1_.get(5) <= 3;
+		return (p_175569_1_.get(Calendar.MONTH) + 1 == Calendar.OCTOBER) && (p_175569_1_.get(Calendar.DAY_OF_MONTH) >= 20)
```

Comment:
This one I think you'll want, as it makes the logic a bit more readable by pulling in named constants.